src/Spago/Command/Ls.purs

```diff
@@ -45,6 +53,18 @@ type LsEnv =
   , selected :: WorkspacePackage
   }
 
+listPaths :: Spago { logOptions :: LogOptions } Unit
+listPaths = do
```

Comment:
```suggestion
listPaths { json } = do
```

We don't want flags for each path, but we do want a flag to output Json, as with the other ls commands